pkg/blockstorage/gcepd/gcepd.go

```diff
@@ -548,7 +548,25 @@ func isNotFoundError(err error) bool {
 }
 
 func (s *gpdStorage) FromRegion(ctx context.Context, region string) ([]string, error) {
```

Comment:
How often is this function called? Does this need to be a one-time thing?

Reply:
It's called every restore I believe. I think that's fine.